fix(docs): maintenance patches on cluster (#7405)
inverse-inc/packetfence: docs/PacketFence_Upgrade_Guide.asciidoc

== Apply maintenance patches

=== Important note for cluster environments

In cluster environments, you need to perform following steps on **one server

> at a time**. To avoid, several moves of virtual IP addresses, you can start by

Reviewer: ```suggestion
at a time**. To avoid multiple moves of the virtual IP addresses, you can start with
```

> node(s) which didn't own any virtual IP addresses first. You must ensure all

Reviewer: ```suggestion
nodes which don't own any virtual IP addresses first. You must ensure all
```

> services have been restarted correctly before moving to next nodes.

Reviewer: ```suggestion
services have been restarted correctly before moving to the next node.
```
